dist/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+        integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+    <title>Zhu's Roast Chicken and Prawn Restaurant</title>
+<script defer src="main.js"></script></head>
+
+<body>
+    <header>
+        <div id="logo-div" class="text-center">
+        </div>
+        <div class="text-center">
+            <h1>Welcome to Dragon Restuaurant!</h1>
+            <h6 id="dish-num"></h6>
+        </div>
+    </header>
```

Comment:
- [ ] Following the project requirements your `Home page header` and navigation should be similar to the given [mockup](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md). Kindly fix that

Here is the design
![Screenshot from 2021-10-21 16-24-58](https://user-images.githubusercontent.com/40503704/138308994-e9c254b6-1161-4714-966b-571327762f75.png)

Here is yours 
![Screenshot from 2021-10-21 16-25-33](https://user-images.githubusercontent.com/40503704/138309091-6ea84425-7b92-4bf1-b3eb-c2c955022d56.png)